voidcrew/code/modules/overmap/ships/pvp/misc.dm

```diff
@@ -0,0 +1,55 @@
+#define SHIP_GRACE_TIME (1 MINUTES)
+
+/obj/structure/overmap/ship/simulated
+	var/obj/machinery/computer/helm/most_recent_helm // For sending messages to the target's helm
+	var/engine_cooldown = 0 // The point in time when your engines will work again
```

Comment:
you should use the cooldown macros for this (it makes managing cooldowns far easier), see `code/__DEFINES/cooldowns.dm` (at the bottom) for more details